tox.ini

```diff
@@ -5,6 +5,7 @@ skip_missing_interpreters = True
 [common]
 deps =
     pytest
+    pytest-container >= 0.4.3
```

Comment:
are we sure we want >= here? it might pick a version that breaks compatibility

Reply:
> are we sure we want >= here? it might pick a version that breaks compatibility

when we notice that, we can introduce a `<` or `!=` condition (or decide to swallow the incompatibility and upgrade